### DISCUSSION TITLE
I know I am going to say to myself DUH but.

### DISCUSSION THREAD

**Discussion by Dentonknifeworks** [aaronwmorris/indi-allsky#851] [Q&A] [answered]
Where do I enable the new smoke overlay?
Thanks
Brad

**Comment by aaronwmorris** [comment, marked as answer]
There is a smoke indicator in the lower right hand side of the web interface.  You can also add it to the image overlay label using the `{smoke_rating:s}` variable.

https://github.com/aaronwmorris/indi-allsky/wiki/Image-Labels  (I just added the example here)
